src/main/java/com/amihaiemil/camel/EvenlyIndentedLine.java

```diff
@@ -75,4 +75,17 @@ public int indentation() {
     public String toString() {
         return this.line.toString();
     }
+
+    @Override
+    public int compareTo(final YamlLine other) {    
```

Comment:
In this method, just ``return this.line.compareTo(other)``, since the class is a decorator :)